modules/local/amber/main.nf

```diff
@@ -0,0 +1,58 @@
+process AMBER {
+    tag "${meta.id}"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hmftools-amber:4.0.1--hdfd78af_0' :
+        'biocontainers/hmftools-amber:4.0.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(tumor_bam), path(normal_bam), path(tumor_bai), path(normal_bai)
+    val genome_ver
+    path heterozygous_sites
+    path target_region_bed
+
+    output:
+    tuple val(meta), path('amber/'), emit: amber_dir
+    path 'versions.yml'            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
```

Comment:
Is there no prefix for this output?

Reply:
For AMBER there are multiple output files written to a directory with one of several prefixes, which depend on execution mode